.github/workflows/pypi-publish.yml

```diff
@@ -5,26 +5,57 @@ name: Publish Python 🐍 distributions 📦 to PyPI
 
 on:
   release:
-    types: [published]
+    types: [released]
```

Comment:
What's the difference here? :eyes:

Reply:
That's for prereleases: https://github.com/packit/specfile/commit/babf6b0c41459b5a4c3d56eabec631c0693665c6